src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,162 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public boolean add(T value) {
-        return false;
+        addToEnd(value);
+        return true;
     }
 
     @Override
     public void add(T value, int index) {
-
+        indexCheckForAdd(index);
+        if (index == size && value != null) {
```

Comment:
why can't we add null value?

Reply:
had a thought that it makes no sense to add an empty value to the list.

Reply:
in List impl in JDK we can work with null values)